packages/phenomic-plugin-renderer-react/src/render/client.js

```diff
@@ -18,11 +20,13 @@ function render(routes: () => React$Element<any>) {
   }
 
   const initialStateNode = document.getElementById("Hydration")
-  const store = createStore(
-    initialStateNode && initialStateNode.textContent
-      ? JSON.parse(initialStateNode.textContent)
-      : undefined,
-  )
+  store =
+    store ||
```

Comment:
This change is required for hot loading to work properly (as we re-render the full app). @bloodyowl can you confirm this won't cause any trouble for static rendering? (as it's in render/client.js ^^),